templates/types/streaming/fastapi/app/api/routers/upload.py

```diff
@@ -14,7 +14,7 @@
 class FileUploadRequest(BaseModel):
     base64: str
     filename: str
-    params: Any
+    params: Any = None
```

Comment:
Just a nit! As this is a model for FastAPI endpoint so we can make it better with:
```suggestion
    params: Optional[Any]= None
```
And add import:  `from typing import Optional` . 
Or we can even describe it explicitly

```suggestion
    params: Optional[Dict] = Field(default=None, description="What does the params mean")
```